AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1176,7 +1175,14 @@ - (BOOL)openURL:(NSURL *)url {
     if (queryTesterAppUpdateSetupFailed) {
       MSLogDebug([MSDistribute logTag], @"In-app updates setup from tester app failure detected.");
       [MS_APP_CENTER_USER_DEFAULTS setObject:queryTesterAppUpdateSetupFailed forKey:kMSTesterAppUpdateSetupFailedKey];
-      [self startUpdateOnStart:YES];
+
```

Comment:
nit: Can be reused?

Reply:
imho it's better than passing extra flag to function like before